src/Shape/Shape.js

```diff
@@ -504,23 +505,32 @@ export default class Shape extends BaseClass {
     this._hoverGroup = elem(`g.d3plus-${this._name}-hover`, {parent: this._group});
     this._activeGroup = elem(`g.d3plus-${this._name}-active`, {parent: this._group});
 
-    const that = this;
-
     const hitAreas = this._group.selectAll(".d3plus-HitArea")
       .data(this._hitArea ? data : [], key);
 
-    hitAreas.order().transition(this._transition)
+    hitAreas.order()
       .call(this._applyTransform.bind(this));
 
-    const hitEnter = hitAreas.enter().append("rect")
+    const isLine = this._name === "Line";
+
+    isLine && this._path
+      .curve(paths[`curve${this._curve.charAt(0).toUpperCase()}${this._curve.slice(1)}`])
+      .defined(this._defined)
+      .x(this._x)
+      .y(this._y);
```

Comment:
can you explain why this is needed here? is it because we are drawing the hitAreas before running the Line's internal draw method?

Reply:
Yes, that's exactly it and the line hitArea won't render without this being set.

Reply:
Makes sense. I'm not a big fan of this solution, as it starts to put too much (IMO) shape specific code into the abstract Shape class... but I don't have a better solution! So let's ship this as-is for now.